source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -850,18 +850,28 @@ are parsed from the ismaster response in the obvious way.
 Server Description Equality
 ```````````````````````````
 
-For the purposes of updating topology description and publishing SDAM events,
-two server descriptions having the same address
-MUST be considered equal if and only if the values of
-`ServerDescription`_ fields marked (=) are respectively equal.
+For the purpose of determining whether to publish SDAM events, two server
+descriptions having the same address MUST be considered equal if and only if
+the values of `ServerDescription`_ fields marked (=) are respectively equal.
+Two topology descriptions MUST be considered equal if and only if they have
+the same set of server addresses, and the server descriptions for each address
+are respectively equal as just defined.
 
 This specification does not prescribe how to compare server descriptions
 with different addresses for equality.
 
-Note: Server description for each server MUST be updated (replaced)
-every heartbeat. However, new description MUST NOT cause the SDAM flow
-to be executed if the new description is equal, as defined in this section,
-to the previous description.
+Server description for each server MUST be updated (replaced) on every
```

Comment:
nit: "The server description"

Reply:
This change is no longer part of this PR.